mapviz/CMakeLists.txt

```diff
@@ -1,14 +1,25 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.10...3.17)
+
+if(${CMAKE_VERSION} VERSION_LESS 3.12)
+    cmake_policy(VERSION ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION})
+endif()
 
 if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
 endif()
 
+if(${CMAKE_VERSION} VERSION_EQUAL "3.11.0" OR ${CMAKE_VERSION} VERSION_GREATER "3.11.0")
+  cmake_policy(SET CMP0072 NEW)
+endif()
+
 if (CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
-project(mapviz)
+project(mapviz 
+  VERSION 2.0.0
```

Comment:
I don't remember seeing a version set here in other ROS projects, is this a normal thing now?  I feel like this could be a little annoying if we have to also update this number every time we make a new release.

Reply:
It's a recommended thing as far as the documentation for Modern CMake goes. However, if other ROS 2 projects don't include it, then we can leave it out. It's an optional parameter.

Reply:
Removed the version numbers